model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -1209,6 +1209,30 @@ message ExternalPayload {
   map<string, string> params = 2;  // Arbitrary extra parameters to pass
 }
 
+// These URNs are used to indicate capabilities of environments that cannot
+// simply be expressed as a component (such as a Coder or PTransform) that this
+// environment understands.
+message StandardProtocols {
+  enum Enum {
+    // Indicates suport for progress reporting via the legacy metric APIs.
+    LEGACY_PROGRESS_REPORTING = 0 [(beam_urn) = "beam:protocol:progress_reporting:v0"];
+
+    // Indicates suport for progress reporting via the new metric APIs.
```

Comment:
nit: should we add a reference to clarify what old/new metrics are ?

Reply:
Good point. Done.